src/index.ts

```diff
@@ -128,7 +128,13 @@ export const load = (siteId: string, opts?: LoadOptions): void => {
     if (opts.spa) tracker.setAttribute('data-spa', opts.spa);
   }
   tracker.onload = flushQueue;
-  firstScript.parentNode.insertBefore(tracker, firstScript);
+  
+  if (firstScript) {
+    firstScript.parentNode.insertBefore(tracker, firstScript);
+  } else {
+    /* Fallback if no existing script tag is found on the page */
+    document.querySelector('head').appendChild(tracker);
```

Comment:
Just to confirm, is `<head>` guaranteed to be in the DOM with SvelteKit? It looks like there's some black magic going on if a script can be executed with no `<script>` tag on the page, so just making sure 😅